cysimdjson/cysimdjson.pyx

```diff
@@ -116,6 +116,13 @@ cdef class JSONArray:
 		return self.Array.size()
 
 
+	def __iter__(JSONArray self):
```

Comment:
Good, good.

Now, the important question: should a (now non-existent) generator object lives in C++ (aka C++ iterator?) to remove indicated iteration complexity?

Reply:
@ateska I guess in JSONArray we do not use iterator, this is hanlded by simdjson.

Reply:
Yes, that's exactly a point - there is an SIMDJSON (C++) iterator that should be used here ;-)

Reply:
@ateska Haha, finally done. :D :D :D